packages/web-components/fast-components/build/generate-open-ui-definition.js

```diff
@@ -0,0 +1,211 @@
+import fs from "fs";
+import path from "path";
+import glob from "glob";
+import chalk from "chalk";
+import Ajv from "ajv";
+import openUISchema from "../src/__test__/component.schema.json";
+import webComponentDefinitionSchema from "@microsoft/fast-tooling/dist/schemas/web-component.schema";
+
+const ajv = new Ajv();
+ajv.addSchema(webComponentDefinitionSchema);
+const validate = ajv.compile(openUISchema);
+const openUIDisplayName = "Open UI";
+const webComponentDisplayName = "Web Component";
+
+function getDefinitionLocations(definitionLocations, definitionKeys) {
+    return definitionLocations.reduce(
+        (accumulator, currentValue) => {
+            const pathItems = currentValue.split("/");
+            const definitionKey = pathItems[pathItems.length - 1].split(".")[0];
+            accumulator[definitionKey] = currentValue;
+            definitionKeys.push(definitionKey);
+    
+            return accumulator;
+        },
+        {}
+    );
+}
+
+/**
+ * This normalizes the Web Component definition so that it is acceptable
+ * in the JSON schema
+ */
+function definitionNormalizer(definition) {
+    return {
+        ...definition,
+        tags: Array.isArray(definition.tags)
+            ? definition.tags.map((tag) => {
+                return {
+                    ...tag,
+                    attributes: Array.isArray(tag.attributes)
+                        ? tag.attributes.map((attribute) => {
+                            return {
+                                ...attribute,
+                                default: typeof attribute.default === "string"
+                                    ? attribute.default
+                                    : attribute.default === void 0
+                                    ? void 0
+                                    : `${attribute.default}`
```

Comment:
Web component definitions must always have strings as attribute values, right now we can specify number and booleans and this aids when converting to a JSON schema that the tooling can use. Here for our exported Open UI schema, we change it back so that it will validate as a string.